PetaPoco.Tests.Integration/Databases/MSAccess/MsAccessQueryTests.cs

```diff
@@ -171,6 +171,199 @@ public override void Query_ForMultiPocoWithPropertyMissingSetMethod_ShouldThrow(
+            order.Status.ShouldBeOneOf(Enum.GetValues(typeof(OrderStatus)).Cast<OrderStatus>().ToArray());
+            order.JoinablePersonId.ShouldNotBe(0);
+            order.CreatedOn.ShouldBeLessThanOrEqualTo(new DateTime(1990, 1, 1, 0, 0, 0, DateTimeKind.Utc));
+            order.CreatedBy.ShouldStartWith("Harry");
+
+            order.JoinablePerson.ShouldNotBeNull();
+            order.JoinablePerson.Id.ShouldNotBe(0);
+            order.JoinablePerson.Name.ShouldStartWith("Peta");
+            order.JoinablePerson.Age.ShouldBe(18);
+        }
+
+        // FIXME: System.Data.OleDb.OleDbException : Characters found after end of SQL statement.
+        [Fact]
+        public override void QueryMultiple_ForMultiResultsSetWithSinglePoco_ShouldReturnValidPocoCollection()
+        {
+            AddOrders(1);
+
+            var pd = PocoData.ForType(typeof(Person), DB.DefaultMapper);
+            var od = PocoData.ForType(typeof(Order), DB.DefaultMapper);
+            var pdName = pd.Columns.Values.First(c => c.PropertyInfo.Name == "Name").ColumnName;
+            var odId = od.Columns.Values.First(c => c.PropertyInfo.Name == "Id").ColumnName;
+
+            /*
+            "SELECT * FROM [Orders] o\r\n                         WHERE o.[Id] = @0;\r\n                         SELECT * FROM [People] p\r\n                         WHERE p.[FullName] = @1;"
+
+            SELECT * FROM [Orders] o
+                         WHERE o.[Id] = @0;
+                         SELECT * FROM [People] p
+                         WHERE p.[FullName] = @1;
+            */
+            var sql = $@"SELECT * FROM {DB.Provider.EscapeTableName(od.TableInfo.TableName)} o
+                         WHERE o.{DB.Provider.EscapeSqlIdentifier(odId)} = @0;
+                         SELECT * FROM {DB.Provider.EscapeTableName(pd.TableInfo.TableName)} p
+                         WHERE p.{DB.Provider.EscapeSqlIdentifier(pdName)} = @1";
+
+            Order order;
+            using (var multi = DB.QueryMultiple(sql, "1", "Peta0"))
+            {
+                order = multi.Read<Order>().First();
+                order.Person = multi.Read<Person>().First();
+            }
+
+            order.PoNumber.ShouldStartWith("PO");
+            order.Status.ShouldBeOneOf(Enum.GetValues(typeof(OrderStatus)).Cast<OrderStatus>().ToArray());
+            order.PersonId.ShouldNotBe(Guid.Empty);
+            order.CreatedOn.ShouldBeLessThanOrEqualTo(new DateTime(1990, 1, 1, 0, 0, 0, DateTimeKind.Utc));
+            order.CreatedBy.ShouldStartWith("Harry");
+
+            order.Person.ShouldNotBeNull();
+            order.Person.Id.ShouldNotBe(Guid.Empty);
+            order.Person.Name.ShouldStartWith("Peta");
+            order.Person.Age.ShouldBe(18);
+        }
+
+        // FIXME: System.Data.OleDb.OleDbException : Characters found after end of SQL statement.
+        [Fact]
+        public override void QueryMultiple_ForMultiResultsSetWithMultiPoco_ShouldReturnValidPocoCollection()
```

Comment:
Not sure what the issue is here...generated SQL is:
```sql
SELECT * FROM [JoinableOrders] o
INNER JOIN [JoinablePeople] p ON p.[Id] = o.[JoinablePersonId]
ORDER BY o.[Id] ASC;
SELECT * FROM [JoinableOrderLines] ol
ORDER BY ol.[JoinableOrderId] ASC;
```

Reply:
Following @Curlack's [comment](https://github.com/CollaboratingPlatypus/PetaPoco/pull/696#issuecomment-1737835299), and some [follow-up](https://stackoverflow.com/questions/1280021/in-ms-access-is-it-possible-to-execute-multiple-queries) [research](https://stackoverflow.com/questions/770614/running-multiple-sql-statements-in-the-one-operation), seems MS Access doesn't accept multiple statements in a single executed command in the context of standalone SQL queries (as supposed to VBA procedure or macro).